src/components/Timer/index.css

```diff
@@ -1,24 +1,7 @@
-/* Selections */
-.white-text {
-  color: antiquewhite;
-}
-
-/* Timer */
-.timer-style {
-  padding: 10px;
-  font-size: 32px;
-  font-weight: 600;
-}
-
 /* General */
 .container {
```

Comment:
I believe this not use anywhere in `Timer/index.jsx`.

Closest is `<Grid container>` at line 176 of `Timer/index.jsx`.
Based on their API https://material-ui.com/api/grid/#css, `<Grid container>` is using global css class `.MuiGrid-container`.